src/pages/Chat/components/ChatItem/chatItemTypes.types.ts

```diff
@@ -0,0 +1,8 @@
+export type ChatItemTypes = {
+  id:number;
+  avatar: string;
+  from_user: string;
+  last_text_message: string;
+  date:string;
+  unread_count:number | null;
```

Comment:
Тут немного съехало форматирование, нет проблема между двоеточием и типом
```suggestion
  unread_count: number | null;
```